.github/workflows/esti.yaml

```diff
@@ -594,7 +594,7 @@ jobs:
           cache: "sbt"
       - name: Build lakeFS HadoopFS
         working-directory: clients/hadoopfs
-        run: mvn -Passembly -DfinalName=client --batch-mode --update-snapshots package -DskipTests
```

Comment:
Why? These tests are run in another place.

Reply:
I would be glad if you can point out exactly where these tests are run

Reply:
There is a job called "Test lakeFS Hadoop FileSystem" in Esti. It should run these. Here, the tests are skipped in order to speed up what comes after.

Reply:
Test lakeFS Hadoop FileSystem AFAIU runs only the lakefsfs_contract tests.
I would appreciate if you can pin point the exact file and line where these tests are executed as I was looking for quite a while and couldn't find it

Reply:
It runs the contract tests but it also runs the default profile which contains the unit tests. I see for example `LakeFSFileSystemPresignedModeTest` running there.